### DISCUSSION TITLE
Speed Break Efficiency Block

### DISCUSSION THREAD

**Discussion by Wyze3306** [CustomiesDevs/Customies#85] [Q&A]
How to have a breaking speed according to the efficiency of the pickaxe.

**Comment by unickorn** [comment]
I'm not an expert in behaviour packs, but I believe this is a bit tricky to get right even in vanilla. Could you link a behaviour pack that adds custom blocks that have this feature so we could look into it?

**Comment by dadodasyra** [comment]
Poke, related to #100
